Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -124,17 +134,37 @@ struct DashboardView: View {
                 EnvironmentText()
             }
             .padding()
-            .alert(isPresented: .notNil(blockingMessage)) {
-                Alert(
-                    title: Text(blockingMessage!),
+        }
+        // Can only use 1 alert() in a View so we combine the different types of Alerts into 1 function.
```

Comment:
Is it you can only use 1 `alert()` or is it only 1 `alert()` would show at a time? because if latter is the case, you can have two alerts and make their display optional on `blockingAlert!.callToActionButton` and only the one with true would show at a time.